nowradio/refresh_locations.py

```diff
@@ -0,0 +1,63 @@
+from dbManager import Queries, get_connection
+import ourUtils
+
+# This batch is used to fill in the longitude and latitude data
+# for every ip in our station_info table. It collects all of our ips
+# and spawns off threads which use a geolocation API (ourUtils.location_from_ip)
+# to find the lat and long values for each ip. 
+# Finally inserts these values in the appropriate rows.
+
+
+def insert_locations_by_ip(conn, location_tuples):
+    for location in location_tuples:
+        data = {
+            'ip': location[0],
+            'latitude': location[1][0],
+            'longitude': location[1][1]
+        }
+        ourUtils.db_quick_query(
+            conn,
+            Queries.insert_location_for_ip,
+            data
+        )
+
+
+def get_location_tuples(ip_list):
+    return [(ip, ourUtils.location_from_ip(ip.split(':')[0])) for ip in ip_list]
```

Comment:
I would prefer if this was not a one liner. 
Also, I think it would make more sense to return a list of dicts like:
{'ip': <ip here>, 'latitude': <latitude here>, 'longitude': <longitude here>}
This would also make insert_locations_by_ip make more sense, rather than dealing with 2d arrays